api/rest-client.ts

```diff
@@ -158,6 +154,11 @@ export class RingRestClient {
         ? this.authOptions.refreshToken
         : undefined
     this.hardwareIdPromise = getHardwareId(this.authOptions.systemId)
+    this.baseSessionMetadata = {
+      api_version: apiVersion,
+      device_model:
+        this.authOptions.controlCenterDisplayName ?? 'ring-client-api',
+    }
```

Comment:
Just curious, why was this moved?

Reply:
initialization order issue in the constructor. it isn't set when it's referenced.